test/test_api/test_call_signatures.py

```diff
@@ -21,6 +21,15 @@ def assert_signature(source, expected_name, expected_index=0, line=None, column=
         return signatures[0]
 
 
+def assert_signature_string(source, line, column, expected_string):
+    signatures = Script(source, line, column).call_signatures()
+
+    assert len(signatures) <= 1
+
+    if signatures:
+        assert signatures[0].call_signature == expected_string
```

Comment:
There should be an 

    else:
        assert expected_string is None

there.

The test below is probably running into this.